requests_mock/contrib/_pytest_plugin.py

```diff
@@ -76,6 +75,8 @@ def requests_mock(request):
     responses for unit testing. See:
     https://requests-mock.readthedocs.io/en/latest/
     """
+    import requests_mock as rm_module
+
```

Comment:
does this cache between function runs? wouldn't it slow things down to reimport every time a mock was created? 

I didn't know about the gevent case, happy to support it. Does it make a difference to do like?

```python 
if rm_module is None: 
    import requests_mock as m
    rm_module = m 
```

Speed isn't a big concern in a test suite, just thinking it's unnecessary for what is kind of an edge case.